deepsensor/model/model.py

```diff
@@ -918,35 +923,43 @@ def stitch_clipped_predictions(patch_preds, patch_overlap, patches_per_row, x1_a
                         """
                         if patch_x2_index[0] == data_x2_index[0]:
                             b_x2_min = 0
+                            b_x2_max = b_x2_max + 1
```

Comment:
If this ends up actually being the solution, we need to comment this to explain why.

Reply:
As per our discussion last time, I think it would be a lot better if we can identify why using a `patch_size` of e.g. `(0.5, 0.5)` leads to non-square patches when looking at the number of pixels.